Description: don't try to process doc sub directory

 - remove declaration of Makefile inside doc subtree
 - GTK_DOC_CHECK cause parse error

Forwarded: no
Last-Update: 2015-08-30

--- a/configure.ac
+++ b/configure.ac
@@ -6,7 +6,6 @@ AC_INIT(viking, 1.6, , viking, http://viking.sf.net/)
 AM_INIT_AUTOMAKE([dist-bzip2 dist-zip subdir-objects])
 dnl AC_CONFIG_SRCDIR([src/main.c])
 AC_CONFIG_HEADERS([src/config.h])
-GTK_DOC_CHECK(1.0)
 
 AC_PROG_CC
 AC_PROG_CC_STDC
@@ -392,11 +389,6 @@ AC_CONFIG_FILES([
 		win32/installer/Makefile
 		win32/installer/pixmaps/Makefile
 		win32/installer/translations/Makefile])
-AM_COND_IF([VIKING_DOC],
-	[AC_CONFIG_FILES([
-		doc/Makefile
-		doc/reference/Makefile
-		doc/examples/Makefile])])
 
 AC_OUTPUT([
 	   viking.spec
